docs/content/en/options-reference.md

```diff
@@ -106,6 +106,13 @@ The app's default locale. Should match code of one of defined `locales`.
 
 When using `prefix_except_default` strategy, URLs for locale specified here won't have a prefix. **It's recommended to set this to some locale** regardless of chosen strategy, as it will be used as a fallback locale when navigating to a non-existent route.
 
+## `sortRoutes` <badge>v6.27.4+</badge>
```

Comment:
The badge should be below the header. Otherwise it's included in the link and in the side menu which looks bad.

Reply:
Also. This would end up in 6.28.0 since it's a new feature.

Reply:
Done!